Sources/Vitamin/Components/Progressbar/VitaminProgressbar.swift

```diff
@@ -12,12 +12,16 @@ public final class VitaminProgressbar: UIView {
         variant: Variant = .circular(size: . medium, style: .empty),
         progressType: ProgressType,
         showTrack: Bool = true,
-        leftLabelText: String? = nil
+        leftLabelText: String? = nil,
+        vitaminAccessibilityLabel: String? = nil
```

Comment:
Like we talk on Slack, I think that you can directly use `accessiblityLabel` and eventually set the default value here.
```suggestion
        accessibilityLabel: String = UIProgressView().accessibilityLabel
```

Reply:
I tried to do that here, but UIProgressView().accessibility can be nil. When I allow nil value, then default value doens't work. 🤔